cmake/IgnCreateDocs.cmake

```diff
@@ -137,7 +145,7 @@ function(ign_create_docs)
     # about the spaces between input files/directories. If you put each cmake
     # variable on a separate line to make this `set` command more readable,
     # then doxygen will not generate the correct/complete output.
-    set(IGNITION_DOXYGEN_INPUT "${IGNITION_DOXYGEN_API_MAINPAGE_MD} ${IGNITION_DOXYGEN_AUTOGENERATED_DOC} ${IGNITION_DOXYGEN_TUTORIALS_DIR} ${IGNITION_DOXYGEN_TUTORIALS_MAINPAGE_MD} ${ign_doxygen_component_input_dirs}")
+    set(IGNITION_DOXYGEN_INPUT "${IGNITION_DOXYGEN_API_MAINPAGE_MD} ${IGNITION_DOXYGEN_AUTOGENERATED_DOC} ${IGNITION_DOXYGEN_TUTORIALS_DIR} ${IGNITION_DOXYGEN_TUTORIALS_MAINPAGE_MD} ${ign_doxygen_component_input_dirs} ${IGNITION_DOXYGEN_ADDITIONAL_INPUT_DIRS}")
```

Comment:
Is there any chance to split this in multiple lines? The line is super long.

Reply:
Yeah that's unfortunate, but based on the comment above the line, it looks like the answer is "no" :cry: